### ISSUE TITLE
Cannot understand technical step.

### ISSUE THREAD

**Issue by giuseppe-coco** [gtshs2/Autorec#5]
Hi!

I cannot understand a technical step. If only the weights related to the observed ratings are updated during backpropagation, this means that the weights related to the ratings of the test set will never be touched.

Is this not a network with extreme overfitting ?

How is it able to generalize ?
